wormhole-connect/src/routes/ntt/platforms/evm/getMessage.ts

```diff
@@ -0,0 +1,108 @@
+import { ChainName, ChainId } from '@wormhole-foundation/wormhole-connect-sdk';
+import { Implementation__factory } from '@certusone/wormhole-sdk/lib/esm/ethers-contracts';
+import {
+  parseWormholeLog,
+  RelayerPayloadId,
+  DeliveryInstruction,
+} from '@certusone/wormhole-sdk/lib/esm/relayer';
+import { ethers } from 'ethers';
+import { hexlify } from 'ethers/lib/utils';
+import { NttRelayingType, UnsignedNttMessage } from 'routes/types';
+import { getNativeVersionOfToken } from 'store/transferInput';
+import { getTokenById, getTokenDecimals } from 'utils';
+import { wh } from 'utils/sdk';
+import { getWormholeLogEvm } from 'utils/vaa';
+import { NttManager__factory } from './abis';
+import {
+  getNttManagerMessageDigest,
+  parseWormholeTransceiverMessage,
+} from 'routes/ntt/utils';
+
+export const getMessageEvm = async (
+  tx: string,
+  chain: ChainName | ChainId,
+): Promise<UnsignedNttMessage> => {
+  const provider = wh.mustGetProvider(chain);
+  const receipt = await provider.getTransactionReceipt(tx);
+  if (!receipt) {
+    throw new Error(`No receipt for tx ${tx} on ${chain}`);
+  }
+  const nttManager = NttManager__factory.connect(receipt.to, provider);
+  const tokenAddress = await nttManager.token();
+  const fromChain = wh.toChainName(chain);
+  const tokenId = {
+    chain: fromChain,
+    address: tokenAddress,
+  };
+  const token = getTokenById(tokenId);
+  if (!token) {
+    throw new Error(`Token ${tokenId} not found`);
+  }
+  const wormholeLog = await getWormholeLogEvm(fromChain, receipt);
+  const parsedWormholeLog =
+    Implementation__factory.createInterface().parseLog(wormholeLog);
+  const relayingInfoEvent = receipt.logs.find(
+    (log) =>
+      log.topics[0] ===
+      '0x375a56c053c4d19a2e3445e97b7a28bf4e908617ce6d766e1e03a9d3f5276271',
```

Comment:
should this go into Connect's internal SDK as a const?

Reply:
Yeah, we could move it to a consts file and name it something descriptive like `RELAYING_INFO_EVENT_TOPIC`